Refactor checkout quick start to include curl (#174)
vippsas/vipps-checkout-api: vipps-checkout-api-quick-start.md

<!-- END_COMMENT -->

## Before you begin

> This document covers the quick steps for getting started with the ePayment API.

Reviewer: ```suggestion
This document covers the quick steps for getting started with the Vipps Checkout API.
```

You must have already signed up as an organization with Vipps MobilePay and have
your test credentials from the merchant portal, as described in the
[Getting started guide](https://developer.vippsmobilepay.com/docs/getting-started).

**Important:** The examples use standard example values that you must change to
use *your* values. This includes API keys, HTTP headers, reference, etc.

> ## Your first Payment

Reviewer: ```suggestion
## Your first Checkout
```

### Step 1 - Setup

<Tabs
defaultValue="curl"
groupId="sdk-choice"
values={[
{label: 'curl', value: 'curl'},
{label: 'Postman', value: 'postman'},
]}>
<TabItem value="postman">

**Please note:** Postman is discontinuing their offline version. Use only your test keys and delete them after testing.
Ensure that your company allows for cloud use before continuing.

If you wish to use Postman, import the following files:

* [Checkout API Postman collection](/tools/vipps-checkout-api-postman-collection.json)
* [Global Postman environment](https://github.com/vippsas/vipps-developers/blob/master/tools/vipps-api-global-postman-environment.json)

In Postman, tweak the environment with your own values (see
[API keys](https://developer.vippsmobilepay.com/docs/common-topics/api-keys/)):

* `client_id` - Merchant key required for getting the access token.
* `client_secret` - Merchant key required for getting the access token.
* `Ocp-Apim-Subscription-Key` - Merchant subscription key.

> * `merchantSerialNumber` - Merchant ID.

Reviewer: ```suggestion
* `Merchant-Serial-Number` - Merchant ID.
```

> * `internationalMobileNumber` - The MSISDN for the test app profile you have received or registered. This is your test mobile number *including* country code.

Reviewer: Does not apply to our api

Author: Fixed.

Reviewer: Should be removed

Author: "internationalMobileNumber" right? It should be gone in the new version now.
